chapter6.Rmd

```diff
@@ -224,8 +224,7 @@ Use the function [`factor()`](http://www.rdocumentation.org/packages/base/functi
 ```{r}
 # Definition of vectors
 planets <- c("Mercury", "Venus", "Earth", "Mars", "Jupiter", "Saturn", "Uranus", "Neptune")
-type <- c("Terrestrial planet", "Terrestrial planet", "Terrestrial planet", 
-          "Terrestrial planet", "Gas giant", "Gas giant", "Gas giant", "Gas giant")
+type <- c("Terrestrial planet", "Terrestrial planet", "Terrestrial planet", "Terrestrial planet", "Gas giant", "Gas giant", "Gas giant", "Gas giant")
```

Comment:
You can actually leave this formatted the way it was. It can enhance readability when you add a line break in the code.